### PR TITLE
[MRG] Doc updates

### PR DESCRIPTION
**Pull request by ntolley**
Creating a PR from @stephanie-r-jones documentation updates.

### CODE REVIEW

File: doc/roadmap.rst [hunk 1]
```diff
@@ -46,7 +46,7 @@ One Year Plan Through 2021
 Modularize HNN code to simplify installation, development and maintenance
 -------------------------------------------------------------------------
 
-Over the past year, we have been cleaning up and re-organizing the
+We are working to cleaning up and re-organizing the
```
Comment (ntolley) [inline doc/roadmap.rst:49]:
```suggestion
We are working on cleaning up and re-organizing the
```

File: doc/roadmap.rst [hunk 2]
```diff
@@ -59,30 +59,43 @@ and plotting (`HNN-core`_).
...
-    renamed HNN-GUI. 
--   Parallelization of the simulation in the ``hnn_core.simulator`` module via a
-    combination of MPI and Joblib.
--   All post-processing data analysis functions will be moved from the HNN repository 
-    into the ``hnn_core.analysis`` module.
--   Extending ``hnn_core.simulator`` module to run batch simulations that enable parameter
-    sweeps.
--   Development of functions in HNN-GUI to enable parameter sweeps via the GUI. 
--   Reorganization of the Network class within the ``hnn_core.simulator`` module 
+-   Following best practice in open-source software design, including continuous integration testing,
```
Comment (ntolley) [inline doc/roadmap.rst:66]:
```suggestion
-   Following best practices in open-source software design, including continuous integration testing, 
```

File: doc/roadmap.rst [hunk 3]
```diff
@@ -59,30 +59,43 @@ and plotting (`HNN-core`_).
...
--   Parallelization of the simulation in the ``hnn_core.simulator`` module via a
-    combination of MPI and Joblib.
--   All post-processing data analysis functions will be moved from the HNN repository 
-    into the ``hnn_core.analysis`` module.
--   Extending ``hnn_core.simulator`` module to run batch simulations that enable parameter
-    sweeps.
--   Development of functions in HNN-GUI to enable parameter sweeps via the GUI. 
--   Reorganization of the Network class within the ``hnn_core.simulator`` module 
+-   Following best practice in open-source software design, including continuous integration testing, 
+    develop HNN-core. HNN-core will contain clean and reorganized code, and separate all components that
```
Comment (ntolley) [inline doc/roadmap.rst:67]:
```suggestion
    to develop HNN-core. HNN-core will contain clean and reorganized code, and separate all components that 
```

File: doc/roadmap.rst [hunk 4]
```diff
@@ -59,30 +59,43 @@ and plotting (`HNN-core`_).
...
+-   Cleaning optimization routines in HNN-GUI to interact with HNN-core 
+-   Testing HNN-GUI tutorials with HNN-Core integration 
+-   Rename HNN to HNN-GUI and release updated version to the community 
+-   Extending HNN-core to run batch simulations that enable parameter sweeps.
+-   Development of functions in HNN-GUI to enable parameter sweeps via the GUI. 
 -   Reorganization of Param.py file within ``hnn_core.simulator`` module to multiple files that 
     contain smaller dictionaries of parameters related to different modules of the code.
     See `gh-104`_ for related discussions.
+-   Creation of two modules in the HNN-core, one with parts of the code that interact with 
+    the NEURON simulation of which (``hnn_core.simulator``), and one tha contains post-processing data
```
Comment (ntolley) [inline doc/roadmap.rst:96]:
```suggestion
    the NEURON simulation of which (``hnn_core.simulator``), and one that contains post-processing data 
```

**Comment by jasmainak** [comment]
The rendering is a bit weird @ntolley : https://1038-168215891-gh.circle-artifacts.com/0/html/roadmap.html

**Comment by jasmainak** [comment]
@ntolley @stephanie-r-jones please merge when you are happy

File: doc/roadmap.rst [hunk 5]
```diff
@@ -59,30 +59,43 @@ and plotting (`HNN-core`_).
...
     See `gh-104`_, `gh-124`_, and `gh-129`_ for related discussions.
+    **COMPLETED FEB 2021** 
+-   Develop initial HNN-core documentation and example simulations following those 
+    detailed in the HNN-GUI tutorials https://jonescompneurolab.github.io/hnn-core/stable/index.html
+   **COMPLETED MARCH 2021** 
+-   First release of HNN-Core 0.1 to the community
+    **COMPLETED MARCH 2021** 
+-   Removal of all simulation code from the HNN repository, which will be replaced with by 
+    importing from HNN-Core
+     **COMPLETED MARCH 2021**
```
Comment (ntolley) [inline doc/roadmap.rst:86]:
```suggestion
```

File: doc/roadmap.rst [hunk 6]
```diff
@@ -59,30 +59,43 @@ and plotting (`HNN-core`_).
...
+   **COMPLETED FEB 2021** 
+-   Reorganization of the Network class within HNN-core module 
     to separate cortical column model from exogenous drive, and optimization routines.
     See `gh-104`_, `gh-124`_, and `gh-129`_ for related discussions.
+    **COMPLETED FEB 2021** 
+-   Develop initial HNN-core documentation and example simulations following those 
+    detailed in the HNN-GUI tutorials https://jonescompneurolab.github.io/hnn-core/stable/index.html
+   **COMPLETED MARCH 2021** 
+-   First release of HNN-Core 0.1 to the community
+    **COMPLETED MARCH 2021**
```
Comment (ntolley) [inline doc/roadmap.rst:83]:
```suggestion
```

File: doc/roadmap.rst [hunk 7]
```diff
@@ -59,30 +59,43 @@ and plotting (`HNN-core`_).
...
-    sweeps.
--   Development of functions in HNN-GUI to enable parameter sweeps via the GUI. 
--   Reorganization of the Network class within the ``hnn_core.simulator`` module 
+-   Following best practices in open-source software design, including continuous integration testing, 
+    to develop HNN-core. HNN-core will contain clean and reorganized code, and separate all components that 
+    interact directly with the NEURON simulator (e.g. cell and network intantiation, external drives, etc..), 
+    from those that pertain to post-processing data analysis and plotting functions (e.g. spectra lanalysis). 
+    **COMPLETED FEB 2021** 
+-   Convert installation procedures to PIP 
+   **COMPLETED FEB 2021**
```
Comment (ntolley) [inline doc/roadmap.rst:72]:
```suggestion
```

File: doc/roadmap.rst [hunk 8]
```diff
@@ -59,30 +59,43 @@ and plotting (`HNN-core`_).
...
--   Reorganization of the Network class within the ``hnn_core.simulator`` module 
+-   Following best practices in open-source software design, including continuous integration testing, 
+    to develop HNN-core. HNN-core will contain clean and reorganized code, and separate all components that 
+    interact directly with the NEURON simulator (e.g. cell and network intantiation, external drives, etc..), 
+    from those that pertain to post-processing data analysis and plotting functions (e.g. spectra lanalysis). 
+    **COMPLETED FEB 2021** 
+-   Convert installation procedures to PIP 
+   **COMPLETED FEB 2021** 
+-   Parallelization of the simulations in HNN-core via a combination of MPI and Joblib.
+   **COMPLETED FEB 2021**
```
Comment (ntolley) [inline doc/roadmap.rst:74]:
```suggestion
```

File: doc/roadmap.rst [hunk 9]
```diff
@@ -46,7 +46,7 @@ One Year Plan Through 2021
 Modularize HNN code to simplify installation, development and maintenance
 -------------------------------------------------------------------------
```
Comment (cjayb) [inline doc/roadmap.rst:47]:
```suggestion
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

See [RST cheat sheet](https://docs.anaconda.com/restructuredtext/index.html), I believe these should be 4th level headings (under "One Year Plan...")

File: doc/roadmap.rst [hunk 10]
```diff
@@ -59,30 +59,39 @@ and plotting (`HNN-core`_).
...
 -   Reorganization of Param.py file within ``hnn_core.simulator`` module to multiple files that 
     contain smaller dictionaries of parameters related to different modules of the code.
     See `gh-104`_ for related discussions.
+-   Creation of two modules in the HNN-core, one with parts of the code that interact with 
+    the NEURON simulation of which (``hnn_core.simulator``), and one that contains post-processing data 
+    analysis and plotting functions (``hnn_core.analysis``).
+
 
 LFP/CSD Simulation, Visualization and Data Comparison
 -----------------------------------------------------
```
Comment (cjayb) [inline doc/roadmap.rst:97]: Also `^`, the same for all others below until "Five-year plan"?

File: doc/roadmap.rst [hunk 11]
```diff
@@ -109,13 +118,13 @@ Parameter estimation is an inherent difficulty in neural model simulation.
...
 approach to parameter estimation, namely Sequential Neural Parameter Estimation (SNPE)
-(Gonçalves et al BioRxiv 2019:838383). We will adapt this method for parameter 
+(Gonçalves et al Elife 2020: DOI: 10.7554/eLife.56261). We will adapt this method for parameter 
 estimation to work with HNN-core, enabling estimation of a distribution of parameters
 that could account for empirical data, and then integrate it into the HNN-GUI with 
 example tutorials for the following tasks:
 
-- Move current ERP parameter estimation code from HNN into ``hnn_core.analysis`` module.
-- Develop code for SNPE parameter estimation and visualization in ``hnn_core.analysis``.
+- Move current ERP parameter estimation code from HNN into ``hnn_core.simulator`` module.
```
Comment (cjayb) [inline doc/roadmap.rst:126]: We don't have a `simulator`-module yet?

File: doc/roadmap.rst [hunk 12]
```diff
@@ -59,30 +59,39 @@ and plotting (`HNN-core`_).
...
+-   Testing HNN-GUI tutorials with HNN-Core integration 
+-   Rename HNN to HNN-GUI and release updated version to the community 
+-   Extending HNN-core to run batch simulations that enable parameter sweeps.
+-   Development of functions in HNN-GUI to enable parameter sweeps via the GUI. 
 -   Reorganization of Param.py file within ``hnn_core.simulator`` module to multiple files that 
     contain smaller dictionaries of parameters related to different modules of the code.
     See `gh-104`_ for related discussions.
+-   Creation of two modules in the HNN-core, one with parts of the code that interact with 
+    the NEURON simulation of which (``hnn_core.simulator``), and one that contains post-processing data 
+    analysis and plotting functions (``hnn_core.analysis``).
```
Comment (cjayb) [inline doc/roadmap.rst:93]: No such thing as `analysis` (or `simulator`)?

**Review by cjayb — APPROVED** [review]

File: doc/roadmap.rst [hunk 13]
```diff
@@ -59,30 +59,39 @@ and plotting (`HNN-core`_).
...
-    sweeps.
--   Development of functions in HNN-GUI to enable parameter sweeps via the GUI. 
--   Reorganization of the Network class within the ``hnn_core.simulator`` module 
+-   Following best practices in open-source software design, including continuous integration testing, 
+    to develop HNN-core. HNN-core will contain clean and reorganized code, and separate all components that 
+    interact directly with the NEURON simulator (e.g. cell and network intantiation, external drives, etc..), 
+    from those that pertain to post-processing data analysis and plotting functions (e.g. spectra lanalysis). 
+    **COMPLETED FEB 2021** 
+-   Convert installation procedures to PIP **COMPLETED FEB 2021** 
+-   Parallelization of the simulations in HNN-core via MPI or Joblib.**COMPLETED SEP 2020**
```
Comment (ntolley) [inline doc/roadmap.rst:72]:
```suggestion
-   Parallelization of the simulations in HNN-core via MPI or Joblib. **COMPLETED SEP 2020** 
```

**Pull request merged**